pkg/process/exec_conf.go

```diff
@@ -214,12 +214,14 @@ func cleanup(cmd *cobra.Command) {
 		defer zap.ReplaceGlobals(logger)()
 		defer zap.RedirectStdLog(logger)()
 
-		// okay now that logging is working, inform about the broken keys
-		for _, key := range brokenKeys {
-			logger.Sugar().Infof("Invalid configuration file key: %s", key)
-		}
-		for _, key := range brokenVals {
-			logger.Sugar().Infof("Invalid configuration file value for key: %s", key)
+		if cmd.Annotations["type"] != "helper" {
+			// okay now that logging is working, inform about the broken keys
+			for _, key := range brokenKeys {
+				logger.Sugar().Infof("Invalid configuration file key: %s", key)
+			}
+			for _, key := range brokenVals {
```

Comment:
I think we still want to print invalid values?

Reply:
yes, you are right